tests/README.md

```diff
@@ -26,6 +26,18 @@ It is done or go without `poetry`,
 
 The second way maybe blocked the some missing dependencies at someday, so the first one is recommended.
 
+# What and how to create a unit test
+One command has a unit test, because one `git-*` command is just do one thing, so we can eat a piece of `git-*` command in one time.
+
+For example,
+
+1. The `git-alias` should has a test suite, so create `test_git_alias.py` in the directory `test`
```

Comment:
```suggestion
1. The `git-alias` should have a test suite, so create `test_git_alias.py` in the directory `test`
```

Reply:
We should rename https://github.com/tj/git-extras/blob/main/tests/test_authors.py?

Reply:
Yes, it should be, update later.

Reply:
Update the typo and the pytest coverage will be update in another PR.